In [3]:
import pandas as pd
import numpy as np


In [4]:
#load the contents of a file into a pandas Dataframe

fb = pd.read_csv('data/facebook.csv')
ms = pd.read_csv('data/microsoft.csv')

In [5]:
print(ms.iloc[0, 0])

2014-12-31


In [6]:
print(type(fb))

<class 'pandas.core.frame.DataFrame'>


In [7]:
fb.head()

Date       Open       High        Low      Close  Adj Close   Volume
0  2014-12-31  20.400000  20.510000  19.990000  20.049999  19.459270  4157500
1  2015-01-02  20.129999  20.280001  19.809999  20.129999  19.536913  2842000
2  2015-01-05  20.129999  20.190001  19.700001  19.790001  19.206934  4948800
3  2015-01-06  19.820000  19.840000  19.170000  19.190001  18.624611  4944100
4  2015-01-07  19.330000  19.500000  19.080000  19.139999  18.576082  8045200

In [8]:
#getting the index of the fb dataframe 
fb.index

RangeIndex(start=0, stop=780, step=1)

In [9]:
#getting the columns of the dataframe
fb.columns

Index(['Date', 'Open', 'High', 'Low', 'Close', 'Adj Close', 'Volume'], dtype='object')

In [10]:
#getting the number of rows and columns
fb.shape

(780, 7)

In [11]:
#summary statistics of fb dataframe
fb.describe()

Open        High         Low       Close   Adj Close  \
count  780.000000  780.000000  780.000000  780.000000  780.000000   
mean    80.212705   81.285654   79.022397   80.264897   79.914215   
std     64.226121   65.048907   63.190963   64.198375   64.327846   
min     19.250000   19.500000   18.940001   19.139999   18.576082   
25%     25.525000   26.085000   24.845000   25.475000   25.134512   
50%     53.379999   54.034999   52.930000   53.420000   53.035403   
75%    113.322502  115.779999  110.297499  113.702501  113.261238   
max    245.770004  249.270004  244.449997  246.850006  246.850006   

             Volume  
count  7.800000e+02  
mean   1.204453e+07  
std    8.221848e+06  
min    1.311200e+06  
25%    7.215200e+06  
50%    9.728700e+06  
75%    1.408885e+07  
max    9.232320e+07

The above function shows that there are 780 entries in the dataset. The mean for open, high, low, close and adj close are tabulated above. 